applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -627,7 +628,8 @@ void KratosStructuralMechanicsApplication::Register() {
     KRATOS_REGISTER_ELEMENT("AxisymUpdatedLagrangianElement2D9N", mAxisymUpdatedLagrangian2D9N)
 
     // Register the spring damper element
-    KRATOS_REGISTER_ELEMENT("SpringDamperElement3D2N", mSpringDamperElement3D2N);
+    KRATOS_REGISTER_ELEMENT("SpringDamperElement2D", mSpringDamperElement2D);
+    KRATOS_REGISTER_ELEMENT("SpringDamperElement3D", mSpringDamperElement3D);
```

Comment:
```suggestion
    KRATOS_REGISTER_ELEMENT("SpringDamperElement2D", mSpringDamperElement2D);
    KRATOS_REGISTER_ELEMENT("SpringDamperElement3D", mSpringDamperElement3D);
    KRATOS_REGISTER_ELEMENT("SpringDamperElement3D2N", mSpringDamperElement3D2N); // DEPRECATED, only here for backward compatibility, should be removed after some time
```

otherwise existing cases are broken